test/integration/aboutAPITests.js

```diff
@@ -12,7 +12,7 @@ describe('API Integration Tests', () =>
 
     before(async () => {
       await testUtils.setupTestUsers()
-      await promisify(server.start)({ apiPort: constants.SERVER_PORTS.apiPort })
+      await promisify(server.start)({ apiPort: SERVER_PORTS.apiPort, apiProtocol: SERVER_PORTS.apiProtocol })
```

Comment:
@de-laz All these references to `apiProtocol` on the constant `SERVER_PORTS` shouldn't exist anymore and should now reference the test.json config.

Reply:
@MattyJ007 Got it